src/main/java/hw4/pages/TableWithPages.java

```diff
@@ -0,0 +1,51 @@
+package hw4.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.List;
+
+public class TableWithPages extends AbstractPage {
+
+    public TableWithPages(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(name = "table-with-pages_length")
+    private WebElement tableEntries;
+
+    @FindBy(xpath = "//option[1]")
```

Comment:
Why do you use index of the element in the `xpath`? What was happen if order change?

Reply:
Fixed, using `Select`